### ISSUE TITLE
Custom Tag List Has Errors

### ISSUE THREAD

**Issue by jamesnogra** [instabot-py/instabot.py#2354]
I am getting this error if I have a custom `tag_list`

```
2019-08-11 03:34:15,478 - testjamesonly - InstaBot - DEBUG - Get Media by tag: asianhunk
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.6/dist-packages/instabot_py/__main__.py", line 307, in <module>
    main()
  File "/usr/local/lib/python3.6/dist-packages/instabot_py/__main__.py", line 303, in main
    bot.mainloop()
  File "/usr/local/lib/python3.6/dist-packages/instabot_py/instabot.py", line 752, in mainloop
    media = medias.pop()
IndexError: pop from empty list
```

This is my tag list:
`tag_list: ["houseforrent", "fliphouse", "bayfronthouse", "simplehouse"]`

**Comment by gabrielecard** [comment]
+1
getting exactly the same errors

**Comment by yurilaaziz** [comment]
thank you for your interest 
fixed in 0.5.8
 please update
